src/apps/ipv4/reassemble.lua

```diff
@@ -1,15 +1,14 @@
 module(..., package.seeall)
 
-local bit = require("bit")
-local constants = require("apps.lwaftr.constants")
-local ctable = require('lib.ctable')
-local ctablew = require('apps.lwaftr.ctable_wrapper')
-local ffi = require('ffi')
-local lwutil = require("apps.lwaftr.lwutil")
-local C = ffi.C
```

Comment:
Just a note.

Isn't it somewhat idiomatically luajit and faster to cache the namespace as it says in http://luajit.org/ext_ffi_tutorial.html#cache ?